CONTRIBUTING.md

```diff
@@ -81,6 +81,10 @@ This repository includes a [development container](https://code.visualstudio.com
 
 The TypeScript repository is relatively large. To save some time, you might want to clone it without the repo's full history using `git clone --depth=1`.
 
+### Filename too long on Windows
+
+You might need to run `git config --system core.longpaths true` before cloning Typescript on Windows.
```

Comment:
TypeScript.

Reply:
On Windows if I try this I get

```
error: could not lock config file C:/Program Files/Git/etc/gitconfig: Permission denied
```

Should it be `--global` instead of `--system`?

Reply:
Both should work (they're just different config locations, both are essentially system-wide - one's just for your user, one is every user) - I guess it depends on how you have it installed as to weather you have rights for the system-wide one.

Reply:
I'll change it to `global`; I got the same error on Windows for `system`